docs/one-click-integrations.md

```diff
@@ -10,13 +10,60 @@ To set up the PagerDuty integration:
 
 1.  On Pulse, click **Integrations** and select **PagerDuty**.
 
-    ![PagerDuty integratio](images/pagerduty.png)
+    ![PagerDuty integration](images/pagerduty.png)
 
 1.  Fill in the necessary details:
 
-    -    **PagerDuty subdomain:** Your custom PagerDuty subdomain. For example, `mysubdomain` if you use the URL `https://mysubdomain.pagerduty.com` to access PagerDuty.
-    -    **Pulse system name:** Name of the [system](https://docs.pulse.codacy.com/#before-you-begin) to associate with the incidents reported by this integration.
+    - **PagerDuty subdomain:** Your custom PagerDuty subdomain. For example, `mysubdomain` if you use the URL `https://mysubdomain.pagerduty.com` to access PagerDuty.
+    - **Pulse system name:** Name of the [system](https://docs.pulse.codacy.com/#before-you-begin) to associate with the incidents reported by this integration.
 
 1.  Follow the instructions on the Pulse UI to create a new webhook on PagerDuty using the provided URL. For example:
 
     ![PagerDuty webhook](images/pagerduty-webhook.png)
+
+## GitHub
+
+Pulse integrates directly with GitHub to receive data about deployments and changes.
+
+!!! important
+    Consider the following before using the GitHub integration:
+
+    -   The integration obtains information about deployments from Git tags that follow the [SemVer](https://semver.org) convention, excluding pre-release versions.
+
+    -   The integration obtains the deployment date from either the creation date of [annotated tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_annotated_tags) or the timestamp when the integration receives the webhook calls for [lightweight tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_lightweight_tags).
+    
+        However, since webhook calls can be delayed, in this case the deployment date on Pulse could be imprecise and impact the metric **Lead time for changes**.
+
+    -   The integration obtains the set of changes that belong to a deployment from the list of commits between the tag of that deployment and the previous tag.
+    
+        Because of this, the integration doesn't consider the first SemVer tag in the repository as a deployment because it's not possible to calculate the list of changes for that deployment.
+
+To set up the GitHub integration:
+
+1.  Make sure that you're creating a Git tag on your repositories for each sucessful deployment to production, or whenever you make a new release available to any user of your application:
+
+    ```bash
+    git tag -a X.Y.Z -m "<Deployment or release message>"
+    ```
+
+    Where X.Y.Z must be a valid SemVer version without pre-release information.
```

Comment:
> > Do you think users could consider changing the way they're using tags to make it possible for the integration to extract all the needed information?
> > I'm thinking if it would be worthwhile to write a sentence or two pointing users in the right direction towards implementing the tag semantics that we're expecting.
> 
> I guess we could add an example, but I am not sure what we could do more. cc\ @DReigada @afsmeira

I made this proposal to include this important bit of information. :+1: 

This first step can be somewhat intimidating, especially if you have lots of repositories and aren't tagging your versions yet. However, since this is essential for the GtiHub integration to work (or at least to be able to calculate all the metrics), I prefer to have this explicit as the very first step of the instructions.